### ISSUE TITLE
reading posts is broken when using a screen reader

### ISSUE THREAD

**Issue by PlasticMemmories** [mormaer/Mlem#268]
**Describe the bug**
What did you do to trigger the bug?
When reading posts with a screen reader, currently the body of the post is read as one long line of text, making it hard to read. This only started happening in the new version of the app.
**Expected Behavior**
What did you expect to happen?
When a post contains multiple lines of text, you should be able to swipe left for the previous line and right for the next line.
**Actual Behavior**
What actually happened?
when you open a post, it reads as "in community by user" and then the body of the post. This is read out all at once
**To Reproduce**
What steps do I need to take to trigger the bug?
1. Go to settings>accessibility>voiceover and turn it on.
2. Open the mlem app and find a post that is broken up into seperate lines
3. Try to read this post with voiceover turned on
**Mlem Information:**
Version + Build number
If this started happening recently, write which version introduced the bug
version 0.1.0.14
**Additional Context**
Any other relevant information
I explained what occurs as well as I could, but I'd like to add that if you haven't used voiceover before, it changes the way you navigate your phone. If you have trouble using voiceover, please comment so I can help you and help get this bug fixed

**Comment by WestonHanners** [comment]
So I got a fix for this, it should read correctly on expanded view (I also threw in some fixes for the interaction buttons)
but its gonna be trickier in list view. meaning when browsing a community it will combine the text, you will have to navigate to the expanded new to get line by line. 

This seems to be a swiftUI limitation, but I will investigate.

**Comment by PlasticMemmories** [comment]
I don't think that linebreaks would be necessary for the posts list, it'd only be good for when a post is open/in focus, at least in my experience as a voiceover user

**Comment by WestonHanners** [comment]
It should be in the next build let me know if you have issues. 

Going to work posting and commenting with screen readers next I think.
